media/src/main/java/com/tealium/media/MediaSession.kt

```diff
@@ -0,0 +1,204 @@
+package com.tealium.media
+
+import com.tealium.media.segments.*
+import java.util.*
+
+interface Session {
+
+    val id: String
+    var state: PlayerState?
```

Comment:
I'd suggest these should be `val` as they should only expect to be modified by the methods declared below.